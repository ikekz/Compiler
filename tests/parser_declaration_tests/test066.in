type
	t = array of a;
begin
end.