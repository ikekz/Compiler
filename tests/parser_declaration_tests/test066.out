(2,15) Error: Identifier not found "a"